bionic/aip/main.py

```diff
@@ -0,0 +1,70 @@
+"""
+This module is run as main in order to execute a task on a worker.
+"""
+
+import logging
+import os
+import sys
+
+from bionic.deps.optdep import import_optional_dependency
+
+
+def run():
+    # Scope the import to this function to avoid raising for anyone not using it.
+    blocks = import_optional_dependency("blocks", raise_on_missing=True)
+    cloudpickle = import_optional_dependency("cloudpickle", raise_on_missing=True)
+
+    ipath = sys.argv[-1]
+    fs = blocks.filesystem.GCSNativeFileSystem()
+    with fs.open(ipath, "rb") as f:
+        task = cloudpickle.load(f)
+
+    # Now that we have the task, setup logging.
+    _setup_logging(task.job_id(), task.config.project)
+    logging.info(f"Read task from {ipath}")
```

Comment:
This log seems redundant since we are printing another log immediately after?

Reply:
Sure, I can remove the next log for clarity.